mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -197,17 +199,23 @@ def apply_monitoring_stream_steps(
             key=mm_schemas.EventFieldType.ENDPOINT_ID,
         )
 
+        # Emits the event in window size of events based on sample_window size (10 by default)
+        graph.add_step(
+            "storey.steps.SampleWindow",
+            name="sample",
+            after="Rename",
+            window_size=self.sample_window,
```

Comment:
as for the `sample_window` it's part of the `EventStreamProcessor` object. You need to remove it from https://github.com/mlrun/mlrun/pull/6261/files#diff-0534b1b9495d5b08cf68d0a32e67a5d32b89c9b96c5796edc8a47d74883b7374R51 and add it as a param to the above `apply_monitoring_stream_steps` function signature.